__tests__/app.test.js

```diff
@@ -0,0 +1,90 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toEqual(expect.any(Array));
+        topics.forEach((topic) => {
+          expect(Object.keys(topic).length).toEqual(2);
+          expect(Object.keys(topic)).toEqual(
+            expect.arrayContaining(["slug", "description"])
+          );
+        });
+        expect(topics).toBeSortedBy("slug", { descending: false });
```

Comment:
This ticket should not deal with topics, queries or anything like that. Please remove these parts, but maybe save it for later as this is related  to a ticket further down the line 😄